app/cypress/rest.py

```diff
@@ -0,0 +1,205 @@
+"""
```

Comment:
Love this addition + the Cypress auth, I'll be sure to move [the utility method](https://github.com/cds-snc/notification-api/pull/2297) from that PR in here too!